README.md

```diff
@@ -142,20 +142,22 @@ This is this link for this project \_https://github.com/peter-akhigbe/New-Awesom
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+- https://github.com/anbehindY/Awesome-books/issues
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project...
+If you like this project, please give a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
+I would like to thank Microverse, my learning partners and every single Youtube channel.
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
```

Comment:
- [ ] [optional} It would be nice if you ensured your MIT link leads to an actual license file and not the license template as it currently does 
![image](https://user-images.githubusercontent.com/61408860/227161395-a4f4e96d-2f4a-4ae0-9821-ff7e5c895819.png)

Reply:
I added the license file to the repository and also updated the readme file